phpBB/language/en/email/post_in_queue.txt

```diff
@@ -1,8 +1,8 @@
-Subject: Post moderation notification - "{TOPIC_TITLE}"
+Subject: Post needs aprroval - "{TOPIC_TITLE}"
```

Comment:
Hi,
There is a typo error on approval.

Reply:
This was merged already so any discovered issues need a ticket in the tracker.